.travis.yml

```diff
@@ -43,9 +43,10 @@ matrix:
     env: TASK=BUILD_EXAMPLES_MAVEN
     os: linux
 
-  - jdk: oraclejdk8
-    env: TASK=BUILD_EXAMPLES_BAZEL
-    os: linux
+  # TODO(songya): re-enable this build once bazel is stable.
+  # - jdk: oraclejdk8
+  #   env: TASK=BUILD_EXAMPLES_BAZEL
+  #   os: linux
```

Comment:
Can we just move it to "allow_failures"?

Reply:
Sure, done.

Reply:
I think you also need to leave the original entry.

Reply:
I see, done.